portal/src/api/api.ts

```diff
@@ -1009,6 +1009,33 @@ class FKApi {
             url: this.baseUrl + "/status",
         });
     }
+
+    public getComments(projectIDOrBookmark): Promise<{posts: []}> {
```

Comment:
TypeScript will let you specify number | string here since you're expecting both. I'd also be fine with two nullable parameters: projectId: number|null, bookmark: string|null, for example.

Reply:
done